### PR TITLE
Remove prerender option from injectRoute

### PR DESCRIPTION
**Pull request by florian-lefebvre**
## Changes

- Removes `prerender` from `injectRoute`
- Removes handling of deprecated `entryPoint`

## Testing

Should still pass

## Docs

https://github.com/withastro/docs/pull/9388

### CODE REVIEW

File: .changeset/two-beans-try.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@astrojs/web-vitals': major
+'astro': major
+---
+
+Remove the prerender option from injectRoute
+
+The `injectRoute` integration method no longer accepts `prerender: true`. Instead the route's entrypoint needs to contain `export const prerender = true`, the same as non-injected routes.
```
Comment (sarah11918) [inline .changeset/two-beans-try.md:8]:
``````suggestion
Removes the `prerender` option from `injectRoute` in the Integrations API.

The `injectRoute` integration method no longer accepts `prerender: true`. 

Instead,  add the `prerender` export to your injected route like you would for other routes defined in the `src/pages` directory:

```astro
---
export const prerender = true
---
```
``````
I have something like this in the upgrade guide. Does this work?

**Review by sarah11918 — COMMENTED** [review]
Looks good @florian-lefebvre - see if what I've got in the integrations guide works for you!

**Review by matthewp — APPROVED** [review]
code looks good

**Review by Fryuni — APPROVED** [review]
Glorious

**Comment by florian-lefebvre** [comment]
The errors are (it concerns both actions and web vitals) caused by the fact that, after building the endpoint from ts to js, `export const prerender` is turned into `const prerender; export { prerender }`. Not sure what we should do about that

**Pull request closed without merging**